infra/modules/providers/azure/storage-account/output.tf

```diff
@@ -0,0 +1,25 @@
+
+output "storage_account_id" {
+  description = "The ID of the storage account."
```

Comment:
Is it possible to ad types to all of these? (note, I'm not 100% sure if types are supported in TF outputs)

Reply:
They are not supported.

![image](https://user-images.githubusercontent.com/10041279/59737111-f2682b80-9221-11e9-9e0a-16509f1aec86.png)